src/util/argParser.ts

```diff
@@ -423,7 +423,7 @@ class ArgParser {
 
       "customBehaviors": {
         describe: "injects a custom behavior file or set of behavior files in a directory",
-        type: ["string"]
+        //type: ["string"]
```

Comment:
Curious why this is commented out, is it causing a typing error? Seeing same for warcInfo above

Reply:
ah, it should be `type: "array"`